src/containers/StudyManage/MemberControl/MemberControl.tsx

```diff
@@ -26,14 +33,147 @@ function MemberControl({
+    setMembers(studyMembers);
+  }, [studyMembers]);
+
+  useEffect(() => {
+    setApplicants(studyApplicants);
+  }, [studyApplicants]);
+
+  useEffect(() => {
+    setMemberNumber(numberOfMembers);
+  }, [numberOfMembers]);
+
+  useEffect(() => {
+    setApplicantNumber(numberOfApplicant);
+  }, [numberOfApplicant]);
+
+  const acceptStudyMember = async (
+    studyGroupId: string,
+    studyGroupMemberId: number,
+  ) => {
+    try {
+      await patchStudyMember(studyGroupId, studyGroupMemberId);
+      dispatch(
+        openAlert({
+          severity: 'success',
+          title: '멤버로 추가되었습니다!',
+          content: '스터디 멤버에 성공적으로 추가되었습니다!',
+        }),
+      );
+    } catch (error) {
+      dispatch(
+        openAlert({
+          severity: 'error',
+          title: '죄송합니다',
+          content: '스터디 멤버 수락에 실패했습니다.',
+        }),
+      );
+      console.error(error);
+      const { response } = error as AxiosError;
+      const { data }: { data: errorType } = response as AxiosResponse;
+      const { errorCode } = data;
+    }
+  };
+
+  const removeStudyMember = async (
+    studyGroupId: string,
+    studyGroupMemberId: number,
+  ) => {
+    try {
+      await deleteStudyMember(studyGroupId, studyGroupMemberId);
+      dispatch(
+        openAlert({
+          severity: 'success',
+          title: '멤버가 제외되었습니다!',
+          content: '스터디 멤버가 성공적으로 제외되었습니다!',
+        }),
+      );
+    } catch (error) {
+      dispatch(
+        openAlert({
+          severity: 'error',
+          title: '죄송합니다',
+          content: '스터디 멤버 제외에 실패했습니다.',
+        }),
+      );
+      console.error(error);
+      const { response } = error as AxiosError;
+      const { data }: { data: errorType } = response as AxiosResponse;
+      const { errorCode } = data;
+    }
+  };
+
+  const deleteMembers = (
+    array: Array<studyManageMemberType>,
+    targetId: number,
+  ) => {
+    const filterArray = array.filter(
+      (applicant) => applicant.studyGroupMemberId !== targetId,
+    );
+    setMembers(filterArray);
+    setMemberNumber((prevState) => (prevState -= 1));
+  };
+
+  const deleteApplicants = (
+    array: Array<studyManagestudyApplicantsType>,
+    targetId: number,
+  ) => {
+    const filterArray = array.filter(
+      (applicant) => applicant.studyGroupMemberId !== targetId,
+    );
+    setApplicants(filterArray);
+    setApplicantNumber((prevState) => (prevState -= 1));
+  };
+
+  const exceptMember = (
+    studyGroupId: string,
+    studyMembers: studyManageMemberType[],
+    memberID: number,
+  ) => {
+    removeStudyMember(studyGroupId, memberID);
+    deleteMembers(studyMembers, memberID);
+  };
+
+  const acceptApplicantToMember = (
+    studyGroupId: string,
+    applicant: studyManagestudyApplicantsType,
+    studyMembers: studyManageMemberType[],
+    applicantID: number,
+    studyApplicants: studyManagestudyApplicantsType[],
+  ) => {
+    acceptStudyMember(studyGroupId, applicantID);
+    const newArray = [
```

Comment:
```suggestion
    const newApplicantsList= [
```
newArray보단 newApplicantsList이 조금 더 직관적인 것 같습니다!

Reply:
추가되는 부분이 멤버라서 newMembersList로 처리했습니다.